src/components/List.js

```diff
@@ -79,6 +80,53 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  const sortedList = docs.sort((a, b) => {
+    const aDaysUntilNextPurchase = getDaysUntilNextPurchase(a);
+    const bDaysUntilNextPurchase = getDaysUntilNextPurchase(b);
+
+    if (!isInactive(a) && isInactive(b)) {
+      return -1;
+    }
+    if (isInactive(a) && !isInactive(b)) {
+      return 1;
+    }
+
+    if (aDaysUntilNextPurchase < bDaysUntilNextPurchase) {
+      return -1;
+    }
+    if (aDaysUntilNextPurchase > bDaysUntilNextPurchase) {
+      return 1;
+    }
+    if (aDaysUntilNextPurchase === bDaysUntilNextPurchase) {
```

Comment:
I think that `if (aDaysUntilNextPurchase === bDaysUntilNextPurchase) {` may be unnecessary. If A is not less than B and A is not greater than B, then A must equal B, right?